Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -247,6 +241,19 @@ private fun processCLib(flavorName: String, cinteropArguments: CInteropArguments
 
     val outKtPkg = fqParts.joinToString(".")
 
+    val jarDirectory = def.config.j2objcJar
```

Comment:
Jar files aren't often specified as a directory containing all jar files they are often specified as individual files ... but this may fit with the rest of the way headers work?

Reply:
Renamed to jarFiles

It's a list of space-seperated values that's in the files. So it would be just pointing to different files. Bad name in hindsight